code/datums/status_effects/_status_effect.dm

```diff
@@ -205,3 +205,8 @@
 /atom/movable/screen/alert/status_effect/Destroy()
 	attached_effect = null //Don't keep a ref now
 	return ..()
+
+/// Based on what's going on with our status effect and the amount of curses we have, update our description.
+/// Only for subtypes
+/atom/movable/screen/alert/status_effect/proc/update_description()
+	SHOULD_CALL_PARENT(FALSE)
```

Comment:
Seems a little weird to have this vs just handling it on the status effect itself

Reply:
it seemed like a cleaner implementation to me to have the status effect handle its own desc rather than keep comingling the desc with the status effect and having to keep track of all these different variables

Reply:
using `update_desc()` is a better idea thanks